Public/Src/Engine/Processes/UnSandboxedProcess.cs

```diff
@@ -301,16 +301,19 @@ protected Process CreateAndSetUpProcess()
         {
             Contract.Requires(Process == null);
 
-            if (!FileUtilities.FileExistsNoFollow(ProcessInfo.FileName))
+            var mode = GetFilePermissionsForFilePath(ProcessInfo.FileName, followSymlink: false);

```

Comment:
>GetFilePermissionsForFilePath [](start = 23, length = 29)

Does this need to be within the #if PLATFORM_OSX?

Reply:
You are right about both, will fix it...